library/src-bootstrapped/scala/quoted/Const.scala

```diff
@@ -23,7 +23,7 @@ object Const {
       case Inlined(_, Nil, e) => rec(e)
       case _  => None
     }
-    rec(expr.unseal)
+    rec(expr.asReflectTree)
```

Comment:
Do we have trees other than reflect trees? What about call it `reflect`?

Reply:
We have other things in `reflect` such as `TypeRepr`, `Position`, `Symbol`, ... . All things that could be taken out from this expression.

There is also a better use of `reflect` in #10289.

We also want the method to be a bit longer to make it pop out a bit bore in the source as this is where we go into the other API.

Reply:
Looking at the changes, for meta-programmers, `asReflectTree` seems a little verbose. What about `expr.asTree`?

Reply:
Perfect, that was the intention. We want to stop anyone that does not need to use it for calling it in the middle of some code just because it has a short name.

Reply:
`asTree` tempted as a first option but I noticed that it was too dangerous and the changed to `asReflectTree` to have something that is immediately associated with `reflect.Tree`.

Reply:
The only realistic alternative we have is to remove this method and create one in the module of `Tree` or `Term`.

```diff
- val tree: Term = expr.asReflectTree
+ val tree: Term = Term.of(expr)
+ val tree: Term = Tree.of(expr)
```
This keeps the short name and has the added advantage that we need to import `qctx.reflect._` or use the path explicitly (making it clear where we change from one API to the other).

Reply:
I opened #10332 with this alternative. This alternative looks better and is more inline with the rest of the API

Reply:
That sounds like a better alternative 👍